lib/upgradedb/mysql.2022072200.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2016 LMS Developers
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ */
+
+$this->BeginTrans();
+
+$this->Execute("ALTER TABLE events ADD COLUMN netnodeid int(11)");
+$this->Execute("ALTER TABLE events ADD CONSTRAINT event_netnodeid_fkey
```

Comment:
brak `DEFAULT NULL`.

Reply:
Czy zdajesz sobie sprawę, że jeśli tego nie dodamy to kolumna nadal będzie miała DEFAULT NULL?

Reply:
postgres i `\d events` daje:

```
...
 ticketid     | integer                |           |          | 
 netnodeid    | integer                |           |          | 
 netdevid     | integer                |           |          | 
...
```

Reply:
@chilek